IDEAS/Buildings/Components/Interfaces/DummyConnection.mo

```diff
@@ -12,12 +14,11 @@ model DummyConnection "Source generator/sink for propsbus"
   parameter Modelica.Units.SI.Temperature T=293.15
     "Fixed temperature for surfRad, or zone when isZone";
 
-  outer IDEAS.BoundaryConditions.SimInfoManager sim
-    annotation (Placement(transformation(extent={{-100,20},{-80,40}})));
+  outer IDEAS.BoundaryConditions.SimInfoManager sim annotation (Placement(transformation(extent={{-100,20},{-80,40}})));
   IDEAS.Buildings.Components.Interfaces.ZoneBus zoneBus(
-    redeclare package Medium = IDEAS.Media.Air,
-    outputAngles=sim.outputAngles,
-    numIncAndAziInBus=sim.numIncAndAziInBus)
+    redeclare package Medium = Medium,
+  numIncAndAziInBus=sim.numIncAndAziInBus,
+    outputAngles=sim.outputAngles, use_port_1 = sim.interZonalAirFlowType <> IDEAS.BoundaryConditions.Types.InterZonalAirFlow.None, use_port_2 = sim.interZonalAirFlowType == IDEAS.BoundaryConditions.Types.InterZonalAirFlow.TwoPorts)
```

Comment:
Could you put `use_port_1` and `use_port_2` on a new line (to improve readability).